com.kavilgroup.gestimator.json

```diff
@@ -0,0 +1,106 @@
+                }
+            ]
+        },
+        {
+            "name": "python3-openpyxl",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} openpyxl"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/22/28/a99c42aea746e18382ad9fb36f64c1c1f04216f41797f2f0fa567da11388/et_xmlfile-1.0.1.tar.gz",
+                    "sha256": "614d9722d572f6246302c4491846d2c393c199cfa4edc9af593437691683335b"
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/3b/d5/181cab9a39dbe8060bd073acae2518e0378e66ff7509c4c6db0881d58e01/jdcal-1.4.tar.gz",
+                    "sha256": "ea0a5067c5f0f50ad4c7bdc80abad3d976604f6fb026b0b3a17a9d84bb9046c9"
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/ba/06/b899c8867518df19e242d8cbc82d4ba210f5ffbeebb7704c695e687ab59c/openpyxl-2.6.2.tar.gz",
+                    "sha256": "1d2af392cef8c8227bd2ac3ebe3a28b25aba74fd4fa473ce106065f0b73bfe2e"
+                }
+            ]
+        },
+        {
+            "name": "python3-peewee",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} peewee"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/c9/e4/f82bd273a7c88e7a2f71652324039c7e382f2f88eced01b9799d6c936ad2/peewee-3.9.3.tar.gz",
+                    "sha256": "603337153536b85c3c5b5e84df4ae0f9cbaff09163a3739451d167e0ad3fe1f7"
+                }
+            ]
+        },
+        {
+            "name": "python3-pyblake2",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} pyblake2"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/a6/ea/559658f48713567276cabe1344a9ef918adcb34a9da417dbf0a2f7477d8e/pyblake2-1.1.2.tar.gz",
+                    "sha256": "5ccc7eb02edb82fafb8adbb90746af71460fbc29aa0f822526fc976dff83e93f"
+                }
+            ]
+        },
+        {
+            "name": "gestimator",
+            "buildsystem": "simple",
+            "cleanup": [ "/Other databases", "/spreadsheets", "/sources", "/screenshots", "/GEstimator.ico", "/pyinstaller_spec.spec", "/installer.nsi", "/build_flatpak.sh"],
+            "build-commands": [
+              "install -D gestimator.py /app/bin/gestimator.py",
```

Comment:
It would be nice if you could install the desktop file, the appdata and the icon using the build system.

Reply:
Desktop, appdata and icon files are installed in lines 91, 92, 93.

Reply:
Yeah, except they are not Flatpak specific. They can stay that way, but you can provide a better experience by using the build system you're using currently as i18n support.

Reply:
I am not sure I understand this. Actually those files are written for flatpak. Can you give any suggessions or links on how to improve. I am new to the whole packaging area.